streams/src/test/java/org/apache/kafka/streams/integration/AdjustStreamThreadCountTest.java

```diff
@@ -466,64 +468,9 @@ public void close() {
                 waitForTransitionFromRebalancingToRunning();
 
                 for (final String log : appender.getMessages()) {
-                    // after we replace the thread there should be two remaining threads with 5 bytes each
-                    if (log.endsWith("Adding StreamThread-3, there are now 3 threads with cache size/max buffer size values as 3/178956970 per thread.")) {
```

Comment:
Apparently there was already a test for the cache being sized correctly after a thread replacement/addition,  I'm guessing this was updated with the incorrect values in the PR that introduced the off by one bug -- probably should have been a red flag if the value of the cache size changed here, not to mention the comment above which explicitly mentions it should be 5 bytes per thread 🙂

Reply:
yeah the comment is off.